chatty-cathy-common/src/main/java/org/common/Message.java

```diff
@@ -0,0 +1,24 @@
+package org.common;
+
+import java.io.Serial;
+import java.io.Serializable;
+
+public class Message implements Serializable {
+    @Serial
+    private static final long serialVersionUID = 1L;
+    private final String messageContent;
+    private final String username;
+
+    public Message(String messageContent, String username) {
+        this.messageContent = messageContent;
+        this.username = username;
+    }
+
+    public String getMessage() {
```

Comment:
Bit pedantic but would it be best to keep to "MessageContent" to keep clear and concise? 
```suggestion
    public String getMessageContent() {
```